user_templates/mcu_defaults/main/Mellow_Fly_Gemini_v3.cfg

```diff
@@ -0,0 +1,51 @@
+
+#------------------------------------------#
+#### CUSTOM TEMPLATE MCU definition ########
+#------------------------------------------#
+
+# This template file is a pre-filled file with Klippain pins alias conventions
+# that can be used if your MCU board is not yet officially supported. Just fill
+# in your MCU pins and you will be good to go :)
+
+[mcu]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+# canbus_uuid: change-me-to-the-correct-canbus-id
+##--------------------------------------------------------------------
+
+[include config/mcu_definitions/main/Mellow_Fly_Gemini_v3.cfg] # Do not remove this line
+[board_pins fly_Gemini_v3_mcu]
+mcu: mcu
+aliases:
+    X_STEP=MCU_DRIVE0_STEP   , X_DIR=MCU_DRIVE0_DIR   , X_ENABLE=MCU_DRIVE0_EN   , X_TMCUART=MCU_DRIVE0_UART   ,
+    Y_STEP=MCU_DRIVE1_STEP   , Y_DIR=MCU_DRIVE1_DIR   , Y_ENABLE=MCU_DRIVE1_EN   , Y_TMCUART=MCU_DRIVE1_UART   ,
+    Z_STEP=MCU_DRIVE2_STEP   , Z_DIR=MCU_DRIVE2_DIR   , Z_ENABLE=MCU_DRIVE2_EN   , Z_TMCUART=MCU_DRIVE2_UART   ,
+
+    E_STEP=MCU_DRIVE3_STEP   , E_DIR=MCU_DRIVE3_DIR   , E_ENABLE=MCU_DRIVE3_EN   , E_TMCUART=MCU_DRIVE3_UART   ,
+
+    DRIVER_SPI_MOSI=MCU_TMC_MOSI , # Used in case of SPI drivers such as TMC2240 or TMC5160
+    DRIVER_SPI_MISO=MCU_TMC_MISO , # Used in case of SPI drivers such as TMC2240 or TMC5160
+    DRIVER_SPI_SCK=MCU_TMC_SCK  , # Used in case of SPI drivers such as TMC2240 or TMC5160
+
+    X_STOP=MCU_DRIVE0_STOP , Y_STOP=MCU_DRIVE1_STOP , Z_STOP=MCU_DRIVE2_STOP ,
+    PROBE_INPUT=MCU_PROBE  ,
+    RUNOUT_SENSOR=MCU_DRIVE3_STOP ,
+
+    E_HEATER=MCU_HEAT0   , E_TEMPERATURE=MCU_TH0   ,
+    BED_HEATER=MCU_BED   , BED_TEMPERATURE=MCU_THB ,
+
+    PART_FAN=MCU_FAN0 , E_FAN=MCU_FAN2 , # MCU_FAN2 is always on
+    CONTROLLER_FAN=MCU_FAN1        ,
+#    EXHAUST_FAN=MCU_FAN           ,
+#    FILTER_FAN=MCU_FAN            ,
+#    HOST_CONTROLLER_FAN=MCU_FAN   ,
+
+#    CHAMBER_TEMPERATURE=MCU_ADC_1 ,
+#    ELECTRICAL_CABINET_TEMPERATURE=MCU_ADC_2 ,
+
+#    LIGHT_OUTPUT=MCU_HEAT1    ,
+#    LIGHT_NEOPIXEL=  ,
+#    STATUS_NEOPIXEL= ,
```

Comment:
Can you uncomment these lines to make them available? This way if people activate one of the feature in Klippain it will not crash due to a missing pin definition. Thanks :)
If there is no pins for them, then just delete the additional lines.

Reply:
Yeah, I commented them out without fully understanding how klippain works just to get myself going, the Gemini v3 seems pretty limited in its pinouts/documentation. I believe that neopixel could probably be enabled, as well as a bus for ADXL, as per siboor's docs, but I have absolutely no clue what the actual pin addresses would be.

When the time comes (for myself) I'll probably do the remapping for FILTER_FAN to one of the mcu's fan headers in overrides (since I'm running an ebb36)